tests/integrations/python/tests/test_google.py

```
@@ -1679,6 +1684,177 @@ class MathProblem(BaseModel):
         print(f"  Confidence: {parsed.confidence}")
 
 
+    # =========================================================================
+    # IMAGE GENERATION TEST CASES
+    # =========================================================================
+
+    @skip_if_no_api_key("google")
+    @pytest.mark.parametrize("provider,model", get_cross_provider_params_for_scenario("image_generation"))
+    def test_41a_image_generation_simple(self, test_config, provider, model):
```

Ruff (0.14.10)

Unused method argument: `test_config`

(ARG002)